.github/workflows/ci.yml

```diff
@@ -334,7 +334,7 @@ jobs:
           path: /tmp/bin
   docker_build:
     name: "Docker build for integration tests"
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Isn't it better to test on older platforms if it's still supported? As I see bootstrap will take care of the cmake requirements anyway.

Reply:
This is only the platform that calls the docker build (the build itself is in alpine), and due to being only docker build we skip the bootstrapping process so its easier to build it on a platform that supports cmake 24 out of the box.